tests/Integration/DataManagementSystem/Test_Client_FTS3.py

```diff
@@ -92,7 +95,7 @@ def test_01_operation(self):
       fileStatusDict[fId] = {'status': 'Finished' if fId % 2 else 'Failed',
                              'error': '' if fId % 2 else 'Tough luck'}
 
-    res = self.client.updateFileStatus(fileStatusDict)
+    res = self.db.updateFileStatus(fileStatusDict)
```

Comment:
Why are you "downgrading" these tests from client to db? These operations are not anymore exposed to the client?

Reply:
No, they are not exposed anymore